src/pe-case-and-asset-map/widget/pe-case-and-asset-map.u-client-script.js

```diff
@@ -0,0 +1,285 @@
+		prefix: 'fa',
+		iconColor: '#fff'
+	};
+	c.awesomeMarkerIconComputer = {
+		type: 'awesomeMarker',
+		icon: 'desktop',
+		markerColor: 'blue',
+		prefix: 'fa',
+		iconColor: '#fff'
+	};
+	c.awesomeMarkerIconConsumable = {
+		type: 'awesomeMarker',
+		icon: 'keyboard',
+		markerColor: 'blue',
+		prefix: 'fa',
+		iconColor: '#fff'
+	};
+	c.awesomeMarkerIconCase = {
+		type: 'awesomeMarker',
+		icon: 'exclamation-circle',
+		markerColor: 'blue',
+		prefix: 'fa',
+		iconColor: '#fff'
+	};
+
+	//Center of the USA
+	c.centerLocation = {
+		lat: 37.3382,
+		lng: 121.8863,
+		zoom: 3,
+		icon: c.awesomeMarkerIconCurrentLocation
+	}
+	c.startLocation = {
+		lat: 37.3382,
+		lng: 121.8863,
+		zoom: 3,
+		icon: c.awesomeMarkerIconCurrentLocation
+	}
+	function getDeviceLocation() {
+		var defer = $q.defer();
+		
+		if(navigator.geolocation && c.liveApp) {
+			navigator.geolocation.getCurrentPosition(function(location) {
+				c.centerLocation.lat = location.coords.latitude;
+				c.centerLocation.lng = location.coords.longitude;
+				c.startLocation.lat = location.coords.latitude;
+				c.startLocation.lng = location.coords.longitude;
+				if(location.coords.accuracy > 14){
+					c.centerLocation.zoom = Math.round(location.coords.accuracy*0.25);			
+					c.startLocation.zoom = Math.round(location.coords.accuracy*0.25);
+				}
+				defer.resolve();
+			});
+		}else{
+			//Default to New York
+			c.centerLocation = {
+				lat: 37.3382,
+				lng: -121.8863,
+				zoom: 9,
+				icon: c.awesomeMarkerIconCurrentLocation
+			};
+			angular.copy(c.centerLocation,c.startLocation);
+			defer.resolve();
+		}
+		return defer.promise;
+	}
+		
+	c.mapbox = {
+		mapbox_layer: {
+			name: 'Mapbox',
+			// url: 'https://api.mapbox.com/styles/v1/mapbox/streets-v10/tiles/256/{z}/{x}/{y}?access_token={apikey}',
+			url: 'https://api.mapbox.com/styles/v1/mapbox/streets-v10/tiles/256/{z}/{x}/{y}?access_token={apikey}',
+			type: 'xyz',
+			options: {
+				apikey: c.data.apiKey,
+				mapid: 'testing'
+			}
+		}
+	};
+	
+	c.$onInit = function() {
+		//debugger;
+		c.searchObject = $location.search();
+
+		c.tile = c.mapbox.mapbox_layer;	
+		getDeviceLocation().then(function(){
+			$timeout(function() {
+				demoMarkers();
+				c.loading = false;	
+			}, 3000);
+		});
+	};
+	
+	$scope.$watch("c.loading", function(value) {
+		if (value === false) {
+			leafletData.getMap().then(function(map) {
+				$timeout(function() {
+					map.invalidateSize();
+				}, 300);
+			});
+		}
+	});
+
+	function demoMarkers(){
+		var list = [];
+		for(var j = 0; j <c.data.list.length; j++){
+			list[j] = {
+				name: c.data.list[j].name,
+				address: c.data.list[j].street +' ' + c.data.list[j].city +', '+ c.data.list[j].state +' '+ c.data.list[j].zip,
+				types: c.data.list[j].types,
+				lat: c.data.list[j].lat,
+				lng: c.data.list[j].lng,
+				sysId: c.data.list[j].sysId,
+				location: c.data.list[j].location
+			};
+
+		}
+		var markers = [];
+		for(var i=0;i<c.data.sites.length;i++){
```

Comment:
Adding simple spaces here would increase readability and show better on the library. Please format based on the ESLint rules for this repo. https://github.com/platform-experience/serviceportal-widget-library/blob/master/.eslintrc.yml

Before:

`for(var i=0;i<c.data.sites.length;i++){`

After:

`for (var i = 0; i < c.data.sites.length; i++) {`

To activate local linting on your widget code, simply run yarn or npm install.